auacm/app/modules/flasknado/flasknado.py

```diff
@@ -0,0 +1,45 @@
+from tornado.websocket import WebSocketHandler
+
+def make_flasknado():
+    """ Creates a new class instance of Flasknado
+
+    In a sense, this is kind of a hack. The problem is that the way Tornado
+    works, it requires sending a WebSocketHandler class rather than an instance
+    of that class. This means that in order to have separate instances, we must
+    dynamically create different classes at runtime. This function simply
+    returns a Flasknado class with an empty set of clients.
+    """
+
+    values = dict(Flasknado.__dict__)
+    values['clients'] = []  # give the new instance a new set of clients
+    return type('Flasknado', Flasknado.__bases__, values)
+
+class Flasknado(WebSocketHandler):
+    """ A class for handling sockets
+
+    This class acts as a singleton that contains all of the websocket clients
+    that are currently connected to the server.
+    """
+
+    clients = []  # initialize the list of clients to an empty list
```

Comment:
Um, not exactly sure how this is being used, but it probably needs to go inside the `emit()` method. The problem is that lists are mutable in Python, so even though they can be accidentally shared across multiple objects if they're defined at the class level.

[Here](https://gist.github.com/BrandonLMorris/12c202a414b70c88a6be) is a quick example I made to demonstrate the issue. I'm not sure if it completely applies since (to my knowledge) we're only using this class for it's static method and not actually creating instances, but I thought you should be aware.